smart_card_connector_app/build/Makefile

```diff
@@ -58,6 +58,10 @@ include $(THIRD_PARTY_DIR_PATH)/pcsc-lite/naclport/server_clients_management/inc
 SOURCES_PATH := ../src
 
 
+JS_COMPILER_FLAGS := \
```

Comment:
nit: I think one empty line between SOURCES_PATH and JS_COMPILER_FLAGS is sufficient